api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-list-account.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "List account"
+description: "Get the userAccountInformations from the account navigation property."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List account
+
+Namespace: Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models
+
+Get the userAccountInformations from the account navigation property.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
+|Delegated (personal Microsoft account)|**TODO: Provide applicable permissions.**|
+|Application|**TODO: Provide applicable permissions.**|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /user/{userId}/profile/account
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required|
+
+## Request body
+Do not supply a request body for this method.
+
+## Response
+If successful, this method returns a `200 OK` response code and a collection of [userAccountInformation](../resources/useraccountinformation.md) objects in the response body.
+
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "get_useraccountinformation"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/user/{userId}/profile/account
+```
+
+### Response
+Note: The response object shown here may be truncated for brevity. All of the properties will be returned from an actual call.
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.fast.pi.viewgenerator.v2.profile.models.useraccountinformation)"
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
+  "value": [
+    {
+      "@odata.type": "#Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models.userAccountInformation",
```

Comment:
Noticed that the properties are not alphabetically ordered in the response.  I suspect this is because it's not ordered in the CSDL.  I'll take a look in resource topic, as there might be some inheritance stuff going on, because it looks partially alphabetized.

Reply:
Just want us to be aware, the published profile API docs now has an update/bug fix PR [7537](https://github.com/microsoftgraph/microsoft-graph-docs/pull/7537) and the profile-list-account.md topic is [here](https://github.com/microsoftgraph/microsoft-graph-docs/pull/7537/files#r412649912).